### PR TITLE
[WIP] Support for upcoming SPT 3.10, using EFT 0.15.2.32757

### PR DESCRIPTION
**Pull request by sailro**
Opening a new PR, given SPT 3.10 is now using 0.15.2.32757

### CODE REVIEW

**Comment by wwangyoushi** [comment]
这个是中文的吗？

File: Properties/Strings.zh-cn.resx
Comment (clevebitr) [inline Properties/Strings.zh-cn.resx:1]: Translated as "自动跟踪愿望单"

**Comment by Kogzu** [comment]
3.10 BE is on 15.5.1.33420 now

**Comment by sailro** [comment]
> 3.10 BE is on 15.5.1.33420 now

Yes thanks I'm going to have a look

**Comment by sailro** [comment]
Closing in favor of https://github.com/sailro/EscapeFromTarkov-Trainer/pull/590

**Pull request closed without merging**